rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -726,4 +728,16 @@ public long getGasEstimationCap() {
     public DbKind databaseKind() {
         return DbKind.ofName(configFromFiles.getString(KeyValueDataSource.KEYVALUE_DATASOURCE_PROP_NAME));
     }
+
+    public long getGasCap() {
+        return configFromFiles.getLong(PROPERTY_RPC_GAS_CAP);
```

Comment:
you may also wanna check if it has the property - same as in `getRpcTimeout()`

Reply:
Addressed